Makefile.toml

```diff
@@ -506,8 +520,56 @@ cargo install \
 '''
 ]
 
+[tasks.delete-infrasys-resources]
+dependencies = [
+   "infra-tools",
+]
+script = [
+'''
+infrasys \ # go through all resources in lock file and delete them 
+   \
+   delete-resources \
+   --lock-config-path "${PUBLISH_INFRA_CONFIG_PATH}" 
+'''
+]
+
+[tasks.check-infra-lock]
+dependencies = ["infra-tools"]
+script_runner = "bash"
+script = [
+'''
+set -e
+
+export PATH="${BUILDSYS_TOOLS_DIR}/bin:${PATH}"
+
+infrasys \
+   --toml-config-path "${PUBLISH_INFRA_CONFIG_PATH}"  \
+   \
+   check-infra-lock \
+   --lock-config-path "${INFRASYS_INFRA_CONFIG_PATH}" \
```

Comment:
I don't think this argument is necessary. We should always keep the toml file and the lock file together in the same directory, so it's safe to assume that. (just like `Cargo.toml` and `Cargo.lock`).